### PR TITLE
Updated the environment variables

### PR DESCRIPTION
**Pull request by SanskritiHarmukh**
- [x] Added the SMTP optional and custom mailer config environment variables.

### CODE REVIEW

File: documentation/self-host/community-edition/install-and-build.mdx [hunk 1]
```diff
@@ -21,10 +21,14 @@ DATABASE_URL=postgresql://username:password@url:5432/dbname # or replace with yo
...
-REFRESH_TOKEN_VALIDITY=604800000
-ACCESS_TOKEN_VALIDITY=86400000
+REFRESH_TOKEN_VALIDITY=604800000 # Default validity is 7 days (604800000 ms) in ms
+ACCESS_TOKEN_VALIDITY=86400000 # Default validity is 1 day (86400000 ms) in ms
 SESSION_SECRET=anothersecretcode123
 
+# Reccomended to be true. Set to false if you are using http
+# Note: Some auth providers may not support http requests and may stop working when set to false
+ALLOW_SECURE_COOKIES=true
+
```
Comment (balub) [inline documentation/self-host/community-edition/install-and-build.mdx:31]: Lets not merge this part just yet, the PR is still open and its not merged.

File: documentation/self-host/community-edition/install-and-build.mdx [hunk 2]
```diff
@@ -50,12 +54,24 @@ MICROSOFT_SCOPE=user.read
...
-MAILER_SMTP_URL=smtps://user@domain.com:pass@smtp.domain.com
-MAILER_ADDRESS_FROM=from@example.com
+MAILER_SMTP_ENABLE=true
+MAILER_USE_CUSTOM_CONFIGS=false
+MAILER_ADDRESS_FROM=<from@example.com>
+
+MAILER_SMTP_URL=smtps://user@domain.com:pass@smtp.domain.com # used if custom mailer configs is false
+
+
+# The following are used if custom mailer configs is true
```
Comment (balub) [inline documentation/self-host/community-edition/install-and-build.mdx:64]: Remove whitespace here.

File: documentation/self-host/community-edition/install-and-build.mdx [hunk 3]
```diff
@@ -84,19 +101,26 @@ Let's understand the major environment variables:
...
-11. `MAILER_ADDRESS_FROM`: The email address that you would be using.
-12. `RATE_LIMIT_TTL`: The time it takes to refresh the maximum number of requests being received.
-13. `RATE_LIMIT_MAX`: The maximum number of requests that Hoppscotch can handle under `RATE_LIMIT_TTL`.
-14. `VITE_BASE_URL`: This is the URL where your deployment will be accessible from.
-15. `VITE_SHORTCODE_BASE_URL`: A URL to generate shortcodes for sharing, can be the same as `VITE_BASE_URL`.
-16. `VITE_BACKEND_GQL_URL`: The URL for GraphQL within the instance.
-17. `VITE_BACKEND_WS_URL`: The URL for WebSockets within the instance.
-18. `VITE_BACKEND_API_URL`: The URL for REST APIs within the instance.
-19. `VITE_APP_TOS_LINK` and `VITE_APP_PRIVACY_POLICY_LINK` are optional and are used to configure the links to the Terms & Conditions and Privacy Policy.
+7. `ALLOW_SECURE_COOKIES`: Determines if cookies should only be sent over secure (HTTPS) connections.
```
Comment (balub) [inline documentation/self-host/community-edition/install-and-build.mdx:104]: Lets put this here after the PR is merged.

**Review by balub — CHANGES_REQUESTED** [review]

File: documentation/self-host/community-edition/install-and-build.mdx (hunk 1 above)
Comment (SanskritiHarmukh) [reply documentation/self-host/community-edition/install-and-build.mdx:31]: Cool

File: documentation/self-host/community-edition/install-and-build.mdx (hunk 2 above)
Comment (SanskritiHarmukh) [reply documentation/self-host/community-edition/install-and-build.mdx:64]: Okay

File: documentation/self-host/community-edition/install-and-build.mdx (hunk 3 above)
Comment (SanskritiHarmukh) [reply documentation/self-host/community-edition/install-and-build.mdx:104]: Got it.

**Review by balub — APPROVED** [review]

**Pull request merged**